### ISSUE TITLE
Some Distros add config files during backend install that can conflict with puppet module

### ISSUE THREAD

**Issue by anthonysomerset** [sensson/puppet-powerdns#51]
For example

installing the bind backend package on ubuntu (16.04 and 18.04) drops in a file to `/etc/powerdns/pdns.d` which is auto included from the default `/etc/powerdns/pdns.conf` file

If you are deploying powerdns with this module this leads to errors like:

`Error: /Stage[main]/Powerdns::Authoritative/Service[pdns]: Systemd restart for pdns failed!
journalctl log for pdns:
-- Logs begin at Thu 2018-05-03 17:03:25 SAST, end at Tue 2018-05-08 22:44:08 SAST. --
May 08 22:44:08 liq-enum-nbo systemd[1]: Stopping PowerDNS Authoritative Server...
May 08 22:44:08 liq-enum-nbo systemd[1]: Stopped PowerDNS Authoritative Server.
May 08 22:44:08 liq-enum-nbo systemd[1]: Starting PowerDNS Authoritative Server...
May 08 22:44:08 liq-enum-nbo pdns_server[17395]: Reading random entropy from '/dev/urandom'
May 08 22:44:08 liq-enum-nbo pdns_server[17395]: Fatal error: Refusing to launch multiple backends with the same name 'bind', verify all 'launch' statements in your configuration
May 08 22:44:08 liq-enum-nbo systemd[1]: pdns.service: Main process exited, code=exited, status=1/FAILURE
May 08 22:44:08 liq-enum-nbo systemd[1]: pdns.service: Failed with result 'exit-code'.
May 08 22:44:08 liq-enum-nbo systemd[1]: Failed to start PowerDNS Authoritative Server.`

the problem can be manually resolved by either deleting the newly created conf file in pdns.d or editing the main conf to comment out or remove the include-dir line so the conf file is not loaded.

I can prepare a PR to resolve this but wanted to open the issue first so the right approach can be agreed upon.

**Comment by ju5t** [comment]
Are you sure this is not related to an issue on your server? #47 mentions a similar problem. 

We include acceptance tests that you could temporarily adjust to test your specific use case. I'll run some tests tomorrow morning myself.

**Comment by anthonysomerset** [comment]
yes it actually is because i'm on bionic so using the os repos at the moment while waiting for powerdns to build a release

**Comment by ju5t** [comment]
It turns out `pdns.d/pdns.simplebind.conf` is added on all Debian-based systems regardless of the source of the installation. This seems to be because of Debian's policies. We'll remove it in the next release to make this module a bit more consistent to other releases.

The reason for not removing `include-dir` yet is because it is a change with much bigger impact on the people using this module who are on Debian-based systems. They may have made configuration changes without `powerdns::config`. This is something for a 2.0.0 release though.
